File: Modul 5. Fungsi/PRAK505/PRAK505-2310817120003-NatalieGraceKatiandagho.py
def Biodata(tahunLahir, nama, asal):
    tahun_sekarang = 2020
    print(f"Perkenalkan Nama Saya: {nama}")
    
    usia = tahun_sekarang - tahunLahir
    print(f"Umur Saya: {usia} tahun")
    print(f"Saya Dari Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari: {asal}")

tahunLahir = int(input())
Namaku = input()
Asal = input()

Biodata(tahunLahir, Namaku, Asal)

File: Modul 5. Fungsi/PRAK505/PRAK505-2310817120003-NatalieGraceKatiandagho.c
#include <stdio.h>

void Biodata(int tahunLahir, char *nama, char *asal)
{
    int tahunSekarang = 2020;
    printf("Perkenalkan Nama Saya: %s\n", nama);

    int usia = tahunSekarang - tahunLahir;
    printf("Umur Saya: %d tahun\n", usia);
    printf("Saya Adalah Angkatan : %d \n", tahunSekarang);
    printf("Asal Saya dari: %s\n", asal);
}

int main()
{
    int tahunLahir;
    char Namaku[20], Asal[15];

    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", Namaku);
    scanf(" %[^\n]%*c", Asal);

    Biodata(tahunLahir, Namaku, Asal);

    return 0;
}
